admin_manual/configuration_user/user_configuration.rst

```diff
@@ -193,7 +193,8 @@ Hover your cursor over their name on the **Users** page until the "..."-menu ico
 appears at the far right. After clicking on it, you will see the **Disable** option.
 
 The user will not longer be able to access their Nextcloud until you enable them again.
-Keep in mind that the files, which were shared by this user will not longer be accessible.
+Keep in mind that the files, which were publicly shared by link by this user will no longer be accessible.
+Files shared internally with other users will stay accessible by those users.
```

Comment:
```suggestion
Files shared internally with other users will stay accessible and internal sharing will still be possible.
```

Reply:
What does that mean?

Reply:
I think the user is still listed in internal sharing so even sharing to it should still be possible?

Reply:
When user is disabled they do not show up in search list when sharing.
But they are shown within Talk.